lib/susedistribution.pm

```diff
@@ -238,6 +238,7 @@ sub init_consoles {
                 password => $testapi::password
             });
         $self->add_console('install-shell', 'tty-console', {tty => 2});
+        $self->add_console('installation',  'tty-console', {tty => check_var('VIDEOMODE', 'text') ? 1 : 7});
```

Comment:
sorry if I didn't make myself clear. I thought of providing a comment in the file :-)